src/app/utils/tests/assert.spec.ts

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable max-lines-per-function */
```

Comment:
not needed ?

Reply:
not yet, mais préférable de l'avoir dans tous les .spec.ts par précaution non ?